topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -0,0 +1,533 @@
+
+We will also apply a log transformation to the cells, as this compresses the variation into a less extreme space, making it easier to see the relative differences between cells. 
+
+After normalising and regressing out unwanted factors, we will then scale the data to have unit variance with a zero mean, so that the mean expression of each gene is not a factor in the analysis and only the variation is, ensuring that the later clustering is driven only by the relative variation of the genes, and not neccesarily how expressive those genes are.
+
+> ### {% icon hint %} Highly Variable Genes?
+>
+> Those of you who are familiar with the [ScanPy Tutorial]() might wonder why we have not reduced the number of genes by performing a highly variable gene selection. 
+> The answer is simply that it did not help with this particular dataset, and that by removing the least variable genes in the analysis, it did help us replicate the analysis in the paper. Try it for yourself as an intermediate step (after this analysis) and see!
+{: .hint}
+
+> ### {% icon hands_on %} Hands-on: Normalize and Scale
+>
+> 1. {% tool [Normalize](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_normalize/scanpy_normalize/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata` (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Method used for normalization"*: `Normalize counts per cell, using 'pp.normalize_total'`
+>        - *"Target sum"*: `10000.0`
+>        - *"Exclude (very) highly expressed genes for the computation of the normalization factor (size factor) for each cell"*: `No`
+>
+> 1. {% tool [Inspect and manipulate](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_inspect/scanpy_inspect/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata_out` (output of **Normalize** {% icon tool %})
+>    - *"Method used for inspecting"*: `Logarithmize the data matrix, using 'pp.log1p'`
+>
+> 1. {% tool [Remove confounders](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_remove_confounders/scanpy_remove_confounders/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata_out` (output of **Inspect and manipulate** {% icon tool %})
+>    - *"Method used for plotting"*: `Regress out unwanted sources of variation, using 'pp.regress_out'`
+>        - *"Keys for observation annotation on which to regress on"*: `total_counts`
+>
+> 1. {% tool [Inspect and manipulate](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_inspect/scanpy_inspect/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata_out` (output of **Remove confounders** {% icon tool %})
+>    - *"Method used for inspecting"*: `Scale data to unit variance and zero mean, using 'pp.scale'`
+>        - *"Maximum value"*: `10.0`
+>
+{: .hands_on}
+
+
+## Dimension Reduction
```

Comment:
dimensionality?

Reply:
I think Dimension is okay

Reply:
both ok, but I think consistency is key here. You've used dimensionality above in some places where if you use 'dimension reduction' here, you could be using dimension above. Maybe consider trying to unify those?